apache-couchdb/recipes/default.rb

```diff
@@ -1,6 +1,6 @@
-include_recipe 'apt::ppa'
+include_recipe 'apt-ies::ppa'
 
-apt_repository 'easybib-ppa' do
+apt_repository 'easybib-couchdb-ppa' do
```

Comment:
@till No, the worst that could happen is a warning by apt that we have duplicate entries in sources.list. The reason for this change is that the current name collides with https://github.com/till/easybib-cookbooks/blob/1b5277062b32471048b51480fcf5bc960be0a16f/apt/recipes/easybib.rb#L4 - if ever those two apt sources diverge, one will overwrite each other, leading to all sorts of race conditions.

I can move this our of that PR though if you want to deploy/merge/test it separate. Your call, since it only affects the couch instances.